app/views/layouts/card_layout.html.haml

```diff
@@ -0,0 +1,29 @@
+
+!!! 5
+%html.no-js(lang="en")
+  %head
+    %meta(charset="utf-8")
+    %meta(http-equiv="X-UA-Compatible" content="IE=edge")
+    %meta(name="viewport" content="width=device-width, initial-scale=1")
+    %meta{:name => "robots", :content => "noindex"}
+    - if Settings.facebook_admins
+      - Settings.facebook_admins.each do |admin_id|
+        %meta(property="fb:admins" content="#{admin_id}")
+    %title
+      - if content_for? :title
+        = yield :title
+        = "— #{Settings.project_name}"
+      - else
+        #{Settings.project_name} — How does your MP vote?
+
+    = stylesheet_link_tag "//code.cdn.mozilla.net/fonts/fira.css"
+    = stylesheet_link_tag "application"
+
+    <!--[if lt IE 9]>
```

Comment:
The IE support is just needed as the only browser that's going to need to see this in the specific browser that's doing the screenshotting